views/celebrities/celebrities.hbs

```diff
@@ -0,0 +1,7 @@
+<h2>Celebrities List</h2>
+
+{{#each celebs}}
+<section>
+        <p>{{this.celeb.name}}</p>
+    </section>
+{{/each}}
```

Comment:
here you are iterating through celebs array. Inside you have an objec, so you want to access its properties directly 

```
  <p>{{name}}</p>
```